src/parse.zig

```diff
@@ -571,7 +578,14 @@ const Parser = struct {
                             break :parse;
                         },
                         .NewLine => return error.UnexpectedToken,
-                        else => {},
+                        .EscapeSeq => {
+                            try node.string_value.append(self.tree.source[tok.start + 1]);
+                        },
+                        else => {
+                            for (self.tree.source[tok.start..tok.end]) |c| {
+                                try node.string_value.append(c);
+                            }
```

Comment:
There is a helper for that actually:

```zig
try node.string_value.appendSlice(self.tree.source[tok.start..tok.end]);
```

Reply:
I was wondering if there was something like that. Thanks!